aten/src/ATen/native/native_functions.yaml

```diff
@@ -1437,6 +1437,13 @@
     CPU, CUDA: native_group_norm_backward
 
 # FFT
+- func: fft.fft(Tensor self) -> Tensor
```

Comment:
what's the plan for the this schema?  It will differ from fft below?  I guess we are okay with mobile because I doubt anyone uses fft on mobile.

Also, we should probably allow namespaces eventually which would make this fft::fft.fft which is a little weird, but whatever.

Reply:
This is temporarily addressed with a naming pattern and manual Python and C++ bindings, analogous to what nn does.